tests/test_gw_repeater.sh

```diff
@@ -111,12 +111,10 @@ main() {
     }
 
     [ "$START_REPEATER" = "true" ] && {
-        index=0
         for repeater in $REPEATER_NAMES; do
             status "Start Repeater (Remote Agent): $repeater"
             "${rootdir}"/tools/docker/run.sh -u "${UNIQUE_ID}" ${VERBOSE_OPT} ${FORCE_OPT} ${RP_EXTRA_OPT} \
-                start-agent -d -n "${repeater}" -m aa:bb:cc:"$index$index" -- "$@"
-            index=$((index+1))
+            start-agent -d "$DELAY" -n "${repeater}" -- "$@"
```

Comment:
Something funky going on here... The `-d` option is for detaching, not for DELAY. And also you removed the indentation.

Same for all other changes in this commit.

Reply:
Oops, yes that's a mistake, thanks.